app/src/main/java/mozilla/lockbox/presenter/EditItemPresenter.kt

```diff
@@ -62,6 +65,12 @@ class EditItemPresenter(
 
         view.isPasswordVisible = false
 
+        dataStore.getUsernamesForDomain(itemId, hostname)
+            .subscribe {
+                view.duplicateList = it
+            }
+            .addTo(compositeDisposable)
```

Comment:
This should probably be structured as to be a subscription on the `get()` above.

e.g. 

```kotlin
val getItem = dataStore.get(itemId)
    .filterNotNull()
    .doOnNext { credentials = it }

getItem
    .map { it.toDetailViewModel() }
    .observeOn(mainThread())
    .subscribe(view::updateItem)
    .addTo(compositeDisposable)

getItem
    .switchMap {
        dataStore.getUsernamesForDomain(it.hostname)
    }
    .subscribe {
        view.duplicateList = it.minus(credentials.username)
    }
    .addTo(compositeDisposable)
```

Not completely happy with the use of `credentials.username` (because we're relying on a side effect of the original `getItem`).